### PR TITLE
Added packages listing

### PR DESCRIPTION
**Pull request by vadi2**
Very early edition, but it's something!

image

### CODE REVIEW

**Comment by Edru2** [comment]
Looks nice!👍

Btw the json page is still needed for the package manager (doesn't need to have a link on the mainpage though)

**Comment by vadi2** [comment]
Right! Thanks.

**Comment by vadi2** [comment]
On second thought I'll change the location back to `/packages`. No need to break it, even in alpha.

File: controllers/packages.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
+local app_helpers = require("lapis.application")
+local validate = require("lapis.validate")
+local capture_errors, assert_error = app_helpers.capture_errors, app_helpers.assert_error
+local lfs = require('lfs')
+local Packages = require("models.packages")
+local Users = require("models.users")
+inspect = require("inspect")
```
Comment (Edru2) [inline controllers/packages.lua]:
```suggestion
```
not sure what this should do, but it's not existing so it's erroring

**Review by Edru2 — APPROVED** [review]
Working now! :tada:

**Review by Kebap — APPROVED** [review]

**Pull request merged**